doc/source/whatsnew/v0.25.0.rst

```diff
@@ -214,7 +214,7 @@ I/O
 - Bug in :func:`read_json` for ``orient='table'`` when it tries to infer dtypes by default, which is not applicable as dtypes are already defined in the JSON schema (:issue:`21345`)
 - Bug in :func:`read_json` for ``orient='table'`` and float index, as it infers index dtype by default, which is not applicable because index dtype is already defined in the JSON schema (:issue:`25433`)
 - Bug in :func:`read_json` for ``orient='table'`` and string of float column names, as it makes a column name type conversion to Timestamp, which is not applicable because column names are already defined in the JSON schema (:issue:`25435`)
--
+- Bug in :func:`json_normalize` for ``errors='ignore'`` and nullable metadata fields, the null values in dataframe were literal nan string and not numpy.nan (:issue:`25468`)
```

Comment:
dataframe --> DataFrame

literal nan string --> the string "na"

double backticks around numpy.nan